lib/ozfarium_web/views/components/icon.ex

```diff
@@ -82,6 +82,11 @@ defmodule OzfariumWeb.HTMLC.Icon do
       "plus_outline" ->
         ~H"""
         <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M12 6v6m0 0v6m0-6h6m-6 0H6" />
+        """
+		
+	    "minus_outline" ->
+		    ~H"""
```

Comment:
Something isn't right with indentation. It should be 2 spaces per level.

Reply:
fixed